tech_sessions/models.md

```diff
@@ -0,0 +1,274 @@
+# Writing Good Models in MVC Frameworks
```

Comment:
Should the title of this be the same as what the user story says?

_Writing Good Models in Rails_

Reply:
We could. Most of this content is applicable in django and ASP.NET MVC also, but I'm happy to change the title.